client/pages/Help.js

```diff
@@ -22,9 +22,16 @@ export default function Help() {
   }
   return (
     <Layout>
```

Comment:
I think this should be your only edit in this PR, unless you add page-specific help options.

(Adding an image to your PR can help reviewers understand what changes you've made)
<img width="1227" alt="Screen Shot 2020-11-15 at 2 37 43 PM" src="https://user-images.githubusercontent.com/57203324/99193550-249bf680-2750-11eb-9e2b-3d1479291a53.png">

I feel like this doesn't really provide much help for students. There are many ways to further develop this page, some of which could be:
1. Embed a snapshot of the targeted website in our website. This doesn't have to be hardcoded either - it could dynamically fetch some information from the URL specified!
2. Question and answer section - could use an Accordion to save space and not overwhelm the user?
3. Describe in more depth what information is at each link
4. Make the help customizable by the specialist
5. Etc.

Obviously some of these options are much more involved than others. But as it is coded right now, viewing the Help page doesn't really answer any questions of mine. Even if one of the links would answer my question, I am unsure which link to click, or where on the link's page to find my answer.

This help page exists to easily answer Google-able questions all in one place. Certain students may not know what questions to Google, or how to use Google. So it's our job to brainstorm questions, and provide easy to view answers.